s29986-PrzemyslawKosiedowski/zajecia1/instalacja_debian_linux.txt

```diff
@@ -0,0 +1,1345 @@
+ext2. Na niej zamontowany jest katalog /boot - zawiera pliki niezbędne do
+rozruchu systemu operacyjnego, takie jak jądro systemowe (kernel), pliki
+konfiguracyjne bootloadera (np. GRUB) oraz inne pliki startowe. 
+
+Po zapisaniu zmian można kontynuować instalację systemu. 
+
+-------------------------------------------------------------------------------
+
+GUIDED – USE ENTIRE DISK AND SET UP ENCRYPTED LVM: tryb, w którym krok po
+kroku prowadzi nas instalator, używamy w nim całej dostępnej przestrzeni
+dyskowej. Pozwala ustawić LVM oraz zaszyfrować system. 
+
+Tak jak wcześniej, najpierw wybieramy dysk. Następnie schemat partycjonowania
+(wybieram ten co wcześniej).  
+
+Przechodzę to konfiguracji LVM, zatwierdzam wybrany schemat. 
+
+Następnie instalator nadpisuje zawartość partycji fizycznej #3 na wybranym
+dysku losowymi (na niej będzie znajdował się LVM) danymi, aby zapobiec
+wyciekom metadanych z zaszyfrowanego woluminu. W każdym momencie można
+przerwać tę operacje, jednakże wpłynie to na jakość szyfrowania.  
+
+Następnym krokiem jest wpisanie hasła, które pozwoli odszyfrować tę partycję.
+Zalecane jest bardzo silne hasło, które zawiera kombinację liter, cyfr i
+symboli, minimum 20 znaków. Jeśli nasze hasło będzie miało mniej niż 8 znaków,
+to instalator zasugeruje użycie silniejszego i zapyta się, czy kontynuować ze
+słabszym hasłem. 
+
+Po wpisaniu hasła instalator przechodzi do konfiguracji LVM. Kroki tej
+konfiguracji opisałem wcześniej. 
+
+
+-------------------------------------------------------------------------------
+
+
+MANUAL - pozwala dokonać podziału partycji ręcznie (z tymi ustawieniami
+zostanę). 
+
+Wybieram mój dysk - SCSI3 (0,0,0) (sda) - 10.7 GB ATA VBOX HARDDISK, po czym
+zezwalam na użycie całego dysku. 
+
+Na początek stworzę partycje fizyczne dla EFI i /boot. W tym celu wybieram
+dostępne miejsce na dysku. 
+
+Z tego miejsca można utworzyć partycję ręcznie, pozwolić instalatorowi zrobić
+to automatycznie (wykorzysta całe dostępne miejsce) lub wyświetlić informacje
+o geometrii dysku (SHOW CYLINDER/HEAD/SECTOR INFORMATION). 
+
+Zamierzam dla każdej z nich wykorzystać po 600MB z dostępnego miejsca. Daję aż
+tyle, żeby mieć rezerwę w razie ewentualnej rozbudowy. 
+
+Pierwszą partycją będzie ESP – partycja dla EFI. Wybieram, by znajdowała się
+na początku wolnego miejsca.  
+
+Następnie pojawia się okno z ustawieniami dla partycji: 
+
+NAME – nazwa partycji. 
+
+USE AS - określa sposób używania partycji. Dostępne opcje: 
+
+Ext4 journaling file system – następca Ext3, oferuje wszystkie funkcje
+poprzednika + m.in. wsparcie dla większych rozmiarów partycji (do 1 EB) i
+plików (pojedynczy plik może mieć nawet 16TB). 
+
+Ext3 journaling file system - następca Ext2, posiada te same cechy co
+poprzednik + obsługuje mechanizm księgowania, który w razie awarii systemu
+pozwoli odzyskać dane. Jest wolniejszy od Ext2. 
+
+Ext2 journaling file system – druga wersja systemu plików Ext; nie korzysta z
```

Comment:
Ext2 _**journaling file system**_ - ... nie korzysta z księgowania...

"this doesn't adds up"